nightly/expensive.txt

```diff
@@ -56,66 +56,65 @@ expensive --timeout=500 near-client near_client tests::consensus::test_consensus
-expensive integration-tests standard_cases tests::standard_cases::rpc::test::test_send_money_over_balance_testnet
-expensive integration-tests standard_cases tests::standard_cases::rpc::test::test_send_money_over_balance_testnet --features nightly_protocol,nightly_protocol_features
-expensive integration-tests standard_cases tests::standard_cases::rpc::test::test_send_money_testnet
-expensive integration-tests standard_cases tests::standard_cases::rpc::test::test_send_money_testnet --features nightly_protocol,nightly_protocol_features
-expensive integration-tests standard_cases tests::standard_cases::rpc::test::test_smart_contract_bad_method_name_testnet
-expensive integration-tests standard_cases tests::standard_cases::rpc::test::test_smart_contract_bad_method_name_testnet --features nightly_protocol,nightly_protocol_features
-expensive integration-tests standard_cases tests::standard_cases::rpc::test::test_smart_contract_empty_method_name_with_no_tokens_testnet
-expensive integration-tests standard_cases tests::standard_cases::rpc::test::test_smart_contract_empty_method_name_with_no_tokens_testnet --features nightly_protocol,nightly_protocol_features
-expensive integration-tests standard_cases tests::standard_cases::rpc::test::test_smart_contract_empty_method_name_with_tokens_testnet
-expensive integration-tests standard_cases tests::standard_cases::rpc::test::test_smart_contract_empty_method_name_with_tokens_testnet --features nightly_protocol,nightly_protocol_features
-expensive integration-tests standard_cases tests::standard_cases::rpc::test::test_smart_contract_self_call_testnet
-expensive integration-tests standard_cases tests::standard_cases::rpc::test::test_smart_contract_self_call_testnet --features nightly_protocol,nightly_protocol_features
-expensive integration-tests standard_cases tests::standard_cases::rpc::test::test_smart_contract_simple_testnet
-expensive integration-tests standard_cases tests::standard_cases::rpc::test::test_smart_contract_simple_testnet --features nightly_protocol,nightly_protocol_features
-expensive integration-tests standard_cases tests::standard_cases::rpc::test::test_smart_contract_with_args_testnet
-expensive integration-tests standard_cases tests::standard_cases::rpc::test::test_smart_contract_with_args_testnet --features nightly_protocol,nightly_protocol_features
-expensive integration-tests standard_cases tests::standard_cases::rpc::test::test_swap_key_testnet
-expensive integration-tests standard_cases tests::standard_cases::rpc::test::test_swap_key_testnet --features nightly_protocol,nightly_protocol_features
-expensive integration-tests standard_cases tests::standard_cases::rpc::test::test_upload_contract_testnet
-expensive integration-tests standard_cases tests::standard_cases::rpc::test::test_upload_contract_testnet --features nightly_protocol,nightly_protocol_features
+expensive integration-tests integration-tests tests::standard_cases::rpc::test::test_access_key_smart_contract_testnet
```

Comment:
It should be `integration_tests` with underscore I think.

Reply:
@mina86 I see, so the 3rd argument is the file name.

Reply:
Yep. As documented in `nightly/README.md`.